css/src/new-dashboard.css

```diff
@@ -1,4 +1,92 @@
 body.toplevel_page_wpseo_dashboard {
+	.notice, .yoast-migrated-notice {
+		display:none;
+	}
+
+	.yoast-new-dashboard-notices:last-child{
+		margin-bottom: 2rem;
+	}
```

Comment:
As per what we discussed, this can't happen here because it would add a permanent margin even when there are no notices, which makes the styling of the page weird.